resources/tag_models/page.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
@lucas42 yup. Whatever your arbitrary decisions about what constitutes a page view it's just a type of event though.